*: remove expression index out of experimental feature (#4034)
pingcap/docs: tidb-configuration-file.md

## experimental

> The `experimental` section describes configurations related to the experimental features of TiDB. This section is introduced since v3.1.0. Currently, it has no related configuration item.

Reviewer: ```suggestion
The `experimental` section, introduced in v3.1.0, describes configurations related to the experimental features of TiDB. Currently, this section has no configuration item.
```
